### ISSUE TITLE
Hardware simulation script

### ISSUE THREAD

**Issue by abeforgit** [12urenloop/Telraam#12]
Script to simulate a reasonably realistic hardware setup. Should generate or play back hardware messages and send them to the telraam instance.

**Comment by ajuvercr** [comment]
Is this like an automated test? Or can the script contain some randomness which would mean a human being would have to verify the correctness.

Could you elaborate on what could go wrong in the communication? I'm thinking about disconnects with the beacons, but that may not happen because of ethernet?
This would simulate the runners and beacons could fail to send a message for a runner passing by?

This is pretty dependant on #13 I guess?

**Comment by abeforgit** [comment]
No randomness in the first instance. We want to test deterministically first, but later we will want randomness. 
Beacons can definitely disconnect, ehternet cables are fragile as heck.
Runners can be missed.
TCP Packets could be malformed.

But most importantly, we want to be able to flood the server with messages to stress test it a bit.

**Comment by abeforgit** [comment]
And no this is not dependent on #13 , the beacons will not interact with the API

Just talked with @mcbloch, leave out the stress testing, this will be a separate issue.
So just a relatively realistic stream of messages. Don't focus too much on testing error scenarios yet, just a happy simulation to start with
